test/test_configs.py

```diff
@@ -436,6 +442,69 @@ def test_redq(self, pixels, independent, continuous):
         qvalue(tensordict)
 
 
+@pytest.mark.skipif(not _has_hydra, reason="No hydra found")
+class TestExplorationConfigs:
+    from torchrl.modules.models.models import _LAYER_CLASS_DICT
+    from torchrl.modules.tensordict_module.exploration import (
+        EGreedyWrapper,
+        AdditiveGaussianWrapper,
+        OrnsteinUhlenbeckProcessWrapper,
+    )
+
+    wrapper_map = {
+        "e_greedy": EGreedyWrapper,
+        "additive_gaussian": AdditiveGaussianWrapper,
+        "ou_process": OrnsteinUhlenbeckProcessWrapper,
+    }
+
+    @pytest.mark.parametrize("network", ["linear", "noisy_linear"])
+    @pytest.mark.parametrize(
+        "wrapper", [None, "e_greedy", "additive_gaussian", "ou_process"]
+    )
+    def test_exploration(self, network, wrapper):
+        additional_config = [
+            "env=cartpole",
```

Comment:
you probably don't want to use `AdditiveGaussian` and OU with cartpole
CartPole has a discrete action space, and those explorations are suited for continuous ones.
Perhaps we could use half-cheetah with these?